modules/administration-guide/pages/removing-user-data.adoc

```diff
@@ -9,141 +9,45 @@
-
-.Authorization token privileges:
-
-* `admin` - Can delete all personal data of all users
-* `user` - Can delete only the data related to the user
-====
-
-* A user or an administrator is logged in the OpenShift cluster with deployed {prod-short}.
-
-* A user ID is obtained. Get the user ID using the commands below:
-
-** For the current user:
-+
-[subs="+quotes,macros,attributes"]
-----
-$ curl -X GET \
-  --header 'Authorization: Bearer __<user-token>__' \
-  'pass:c,m,a,q[+https:+//__<{prod-host}>__]/api/user'
-----
- 
-** To find a user by name: 
-+
-[subs="+quotes,macros,attributes"]
-----
-$ curl -X GET \
-  --header 'Authorization: Bearer __<user-token>__' \
-  'pass:c,m,a,q[+https:+//__<{prod-host}>__]/api/user/find?name=__<username>__'
-----
-
-.Example of obtaining a user ID
+* An active session with administrative permissions to {prod-short}. See xref:configuring-the-administrative-user.adoc[].
 
-This example uses `vparfono` as a local user name.
-
-====
-[subs="+quotes,macros,attributes"]
-----
-$ curl -X GET \
-  --header 'Authorization: Bearer __<user-token>__' \
-  'https://che-vp-che.apps.che-dev.x6e0.p1.openshiftapps.com/api/user/find?name=vparfono'
-----
-
-The user ID is at the bottom of the curl command output.
-
-[source,json]
-----
-{
- "name": "vparfono",
- "links": [
-   {
-.
-.
-.
-   }
- ],
- "email": "vparfono@redhat.com",
- "id": "921b6f33-2657-407e-93a6-fb14cf2329ce"
-}
-----
-====
+* An active `oc` session with administrative permissions to the OpenShift cluster. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the OpenShift CLI].
 
 .Procedure
 
-* Remove the data using the API:
-+
-[subs="+quotes,macros,attributes"]
-----
-$ curl -i -X DELETE \
-  --header 'Authorization: Bearer __<user-token>__' \
-  pass:c,m,a,q[+https:+//__<{prod-host}>__]/api/user/__<user-id>__
-----
+. Get the __<username>__ user __<id>__ `id`: navigate to pass:c,m,a,q[+https:+//__<{prod-host}>__]/swagger/#/user/find_1, click btn:[Try it out], set *name*: __<username>__, and click btn:[Execute]. Scroll down the *Response body* to find the `id` value.
+// +
+// [subs="+quotes,macros,attributes"]
+// ----
+// $ curl -X GET \
+//   --cookie '_oauth_proxy=__<token>__' \
+//   'pass:c,m,a,q[+https:+//__<{prod-host}>__]/api/user/find?name=__<username>__'
+// ----
 
-[NOTE]
-====
-The procedure can be also executed using the Swagger UI available on pass:c,m,a,q[+https:+//__<{prod-host}>__]/swagger
-====
+. Remove the __<id>__ user data that {prod-short} server manages, such as user preferences: navigate to pass:c,m,a,q[+https:+//__<{prod-host}>__]/swagger/#/user/remove, click btn:[Try it out], set *id*: __<id>__, and click btn:[Execute]. Expect a `204` response code:
```

Comment:
To remove the __<id>__ user data that {prod-short} server manages, such as user preferences: navigate to pass:c,m,a,q[+https:+//__<{prod-host}>__]/swagger/#/user/remove, click btn:[Try it out], set *id*: __<id>__, and click btn:[Execute]. Expect a `204` response code: